src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/FileController.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers
 {
+    [ApiExplorerSettings(IgnoreApi=true)]
```

Comment:
What's this in aid of?

Reply:
Its to stop Swagger documenting the standard controllers - which it seems to want to